documentation/concepts/check.md

```diff
@@ -4,25 +4,65 @@ menu: check
 subtitle: Check
 ---
 # Check
-{% include enterprise.html %}
+**The `check` command is currently in beta, and only available in the Enterprise edition of Flyway.**
+<br>{% include enterprise.html %}
```

Comment:
Should be able to replace the `<br>` with a blank line, which would remove a bit more HTML from this file

Reply:
So it does - sorted